scripts/system/more/app-more.js

```diff
@@ -0,0 +1,136 @@
+"use strict";
+
+//  app-more.js
+//  VERSION 1.0
+//
+//  Created by Keb Helion, February 2020.
+//  Copyright 2020 Project Athena and contributors.
+//
+//  This script adds a "More Apps" selector to "Project Athena" to allow the user to add optional functionalities to the tablet.
+//  This application has been designed to work directly from the Github repository.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//	
+(function() {
+	var ROOT = Script.resolvePath('').split("app-more.js")[0];
```

Comment:
Should use spaces instead of tabs. (If don't fix now, add an issue in the community-apps repo.)
Also fix up indenting.